src/components/preview/preview.pw.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import PreviewComponent from "./components.test-pw";
+import Preview from "../../../src/components/preview";
+import {
+  previewComponent,
+  lineComponent,
+} from "../../../playwright/components/preview";
+import { CHARACTERS } from "../../../cypress/support/component-helper/constants";
+import { dlsRoot } from "../../../playwright/components/index";
+import {
+  assertCssValueIsApproximately,
+  checkAccessibility,
+} from "../../../playwright/support/helper";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const pixelsData = [256, 275, 300];
+
+test.describe("check Preview component properties", () => {
+  [...pixelsData].forEach((height) => {
+    test(`should check height as ${height}px for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PreviewComponent height={`${height}px`} />);
+
+      await assertCssValueIsApproximately(
+        previewComponent(page),
+        "height",
+        height
+      );
+    });
+  });
+
+  [...pixelsData].forEach((width) => {
+    test(`should check width as ${width}px for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PreviewComponent width={`${width}px`} />);
+
+      await assertCssValueIsApproximately(
+        previewComponent(page),
+        "width",
+        width
+      );
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check children as ${children} for Preview component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<Preview>{children}</Preview>);
+
+      await expect(dlsRoot(page)).toHaveText(children);
+    });
+  });
+
+  test("should check when loading is True for Preview component", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<PreviewComponent loading />);
+
+    await expect(previewComponent(page)).toBeVisible();
+  });
+
+  test("should check when loading is False for Preview component", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<PreviewComponent loading={false} />);
+
+    await expect(previewComponent(page)).not.toBeVisible();
+  });
+
+  [5, 6, 8, 10].forEach((line) => {
```

Comment:
**suggestion** (non-blocking): instead of this you could create a const at the top of the file like so `const lines = [5, 6, 8, 10]` and these use it here and in the accessibility test